README.md

```diff
@@ -3,4 +3,7 @@ firstrunwizard
 
 A first run wizard that explains the usage of ownCloud to new users
 
-[![Build Status](https://drone.owncloud.com/api/badges/owncloud/firstrunwizard/status.svg?branch=master)](https://drone.owncloud.com/owncloud/firstrunwizard) [![codecov](https://codecov.io/gh/owncloud/firstrunwizard/branch/master/graph/badge.svg)](https://codecov.io/gh/owncloud/firstrunwizard)
+[![Build Status](https://drone.owncloud.com/api/badges/owncloud/firstrunwizard/status.svg?branch=master)](https://drone.owncloud.com/owncloud/firstrunwizard)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=owncloud_admin_audit&metric=alert_status&token=27561a290390636bda4680d9711773dfd64159bc)](https://sonarcloud.io/dashboard?id=owncloud_admin_audit)
```

Comment:
This is firstrunwizrd repo, but there are references to admin_audit

Reply:
Done